CHANGELOG.md

```diff
@@ -9,14 +9,16 @@
 ## 2021-04-15 -- v0.5.6
 ### Added
 - Added `date_created` and `date_modified` indexes to the `records` table
-### Fixed
+- Scheduled tests-regression.yaml to run every Monday 12 AM EST
```

Comment:
There's been a release since this was updated. Could you merge in the latest changes from `main` and make this part of the `v0.5.7` release?

Reply:
I can do that. Good callout!